includes/managers/image.php

```diff
@@ -130,6 +148,21 @@ public function get_lightbox_image_attributes( $id ) {
 		return $attributes;
 	}
 
+	private function delete_custom_images( $post_id ) {
+		$image_meta = wp_get_attachment_metadata( $post_id );
+
+		( new Collection( $image_meta['sizes'] ) )
+			->filter( function ( $value, $key ) {
+				return ( 0 === strpos( $key, 'elementor_custom_' ) );
```

Comment:
Maybe you should use the same regex as above, just to be strict? IDK

Reply:
No, the regex is used there for retrieving the dimensions into variables etc...